app/src/main/res/values/strings.xml

```diff
@@ -281,6 +281,21 @@
         <item>Ingredients</item>
     </string-array>
 
+    <string-array name="fields_array">
```

Comment:
is it meant to be translatable ?

Reply:
No. It will be same for all the languages. I'll fix it.